src/components/actions/filters/ItemFilter.tsx

```diff
@@ -5,15 +5,29 @@ export interface IItemFilterProps extends React.ClassAttributes<ItemFilter> {
   label: string;
   item: string;
   onClear: () => void;
+  crop?: number;
 }
 
+export const ELLIPSIS: string = '...';
+
 export class ItemFilter extends React.Component<IItemFilterProps, any> {
 
   render() {
+    let itemFilter: string = this.props.item;
+    if (this.props.crop) {
+      let itemFilterLength: number = itemFilter.length;
+
+      if (itemFilterLength > Math.abs(this.props.crop)) {
+        itemFilter = this.props.crop > 0
+          ? itemFilter.substring(0, this.props.crop) + ELLIPSIS
```

Comment:
You can use s.truncate(itemFilter, this.props.crop) with underscore strings, just another way to add the ellipsis. We can add third param to custom the ellipsis if we want :) but default is ...